src/main/scala/rocketchip/BaseTop.scala

```diff
@@ -67,7 +67,8 @@ abstract class BaseTopModule[+L <: BaseTop, +B <: BaseTopBundle](
   val io: B = b
 
   val coreplex = p(BuildCoreplex)(outer.c, p)
-  val coreplexIO = coreplex.io
+  val coreplexIO = Wire(new BaseCoreplexBundle(outer.c)(p){})
+  val baseCoreplexIO = coreplex.io
```

Comment:
I would come up with a better name than baseCoreplexIO.  There's already a BaseCoreplex module, but in this case, you're not referring to that.